code/__HELPERS/icons.dm

```diff
@@ -731,10 +731,16 @@ The _flatIcons list is a cache for generated icon files.
 
 	var/icon/add // Icon of overlay being added
 
-		// Current dimensions of flattened icon
-	var/{flatX1=1;flatX2=flat.Width();flatY1=1;flatY2=flat.Height()}
-		// Dimensions of overlay being added
-	var/{addX1;addX2;addY1;addY2}
+	// Current dimensions of flattened icon
+	var/flatX1=1

```

Comment:
This PR also contains the commit from my compiler fixes, but that should go away as soon as that PR is merged. It just made my life easier.